backend/src/v5/models/teamspaces.js

```diff
@@ -155,10 +156,22 @@ Teamspace.getAllTeamspacesWithActiveLicenses = (projection) => {
 };
 
 Teamspace.createTeamspaceSettings = async (teamspace) => {
-	const settings = { _id: teamspace, topicTypes: DEFAULT_TOPIC_TYPES, riskCategories: DEFAULT_RISK_CATEGORIES };
+	const settings = { _id: teamspace,
+		topicTypes: DEFAULT_TOPIC_TYPES,
+		riskCategories: DEFAULT_RISK_CATEGORIES,
+		permissions: [{ user: teamspace, permissions: [TEAMSPACE_ADMIN] }] };
```

Comment:
Can we break this up.

Have another function to assign the admin permission and whatever is calling `createTeamspaceSettings` should call that as well

There is too much assumptions here, in the future we may be creating teamspaces where the owner is not named the same as the teamspace.